.gitignore

```diff
@@ -1,3 +1,4 @@
+_version.py
```

Comment:
What is this file and why explicitly ignored?

Reply:
This is used by `scm` and you do not place it under source code control.